src/connector/src/source/base.rs

```diff
@@ -107,6 +107,8 @@ pub trait SplitReaderV2: Sized {
         properties: Self::Properties,
         state: Vec<SplitImpl>,
         parser_config: ParserConfig,
+        metrics: Arc<SourceMetrics>,
+        source_context: SourceInfo,
```

Comment:
The variable name may need renaming as well 🤔 (ditto for all other variables)

Reply:
Done, thanks for reminding!